exercises/practice/simple-linked-list/SimpleLinkedListTests.cs

```diff
@@ -1,83 +1,65 @@
 using System.Linq;
 using Xunit;
+using System.Collections.Generic;
 
 public class SimpleLinkedListTests
 {
     [Fact]
     public void Single_item_list_value()
     {
         var list = new SimpleLinkedList<int>(1);
-        Assert.Equal(1, list.Value);
+        Assert.Equal(1, list.Remove());
     }
 
-    [Fact(Skip = "Remove this Skip property to run this test")]
-    public void Single_item_list_has_no_next_item()
-    {
-        var list = new SimpleLinkedList<int>(1);
-        Assert.Null(list.Next);
-    }
-
-    [Fact(Skip = "Remove this Skip property to run this test")]
+    [Fact]
     public void Two_item_list_first_value()
     {
-        var list = new SimpleLinkedList<int>(2).Add(1);
-        Assert.Equal(2, list.Value);
+        var list = new SimpleLinkedList<int>(1, 2);
```

Comment:
I would like to build this exercise more with more granular steps, perhaps using `Add`/`Count` or `Add`/`Remove` first. Definitely Add makes sense to start with. 

The constructors are, in my opinion, very idiomatic for .net so it would be good to keep them, but they could be a separate step with some narrative to help understand why. 

I'll propose something soon (I hope, if not then next week as I will be mostly offline Thursday to Sunday).

Reply:
I think the tests should follow the TDD style, and with steps they can. The C# track already does it so much better than many other tracks I have tried, and with reflection at our disposal we can easily make it work great leading the student through the exercise step by step. 

If you don't mind @ErikSchierboom, I'd like to give it a go, implement something like this so we can talk specific pros and cons on an example.